data-config.json

```diff
@@ -2,6 +2,7 @@
     "project-settings": {
```

Comment:
In testing, a unit test class was created for my top-level model class, but for the composite multi-field child model (DemoLink in the case of the example config file) the unit test class was not generated.

Reply:
I did that on purpose. I think child model(DemoLink) can be tested and covered within the top-level class while testing the DemoLink List object.

Do let me know if I am missing something here. I would be happy to make necessary changes.

Reply:
I think the DemoLink model should have its own tests rather than being covered by parent tests, as a child model (such as a generic Link class) can be used by many parent model objects (which is why the `use-existing-model` exists)